components/prize_pool/wikis/fighters/prize_pool_custom.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=fighters
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Variables = require('Module:Variables')
+
+local OpponentLibrary = require('Module:OpponentLibraries')
+local Opponent = OpponentLibrary.Opponent
+
+local PrizePool = Lua.import('Module:PrizePool')
+
+local LpdbInjector = Lua.import('Module:Lpdb/Injector')
+local CustomLpdbInjector = Class.new(LpdbInjector)
+
+local CustomPrizePool = {}
+
+local PRIZE_TYPE_POINTS = 'POINTS'
+local TIER_VALUE = {8, 4, 2}
+
+-- Template entry point
+---@param frame Frame
+---@return Html
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	args.syncPlayers = true
+	args.import = true
+
+	local prizePool = PrizePool(args)
+
+	return prizePool:create():setLpdbInjector(CustomLpdbInjector()):build()
+end
+
+---@param lpdbData placement
+---@param placement PrizePoolPlacement
+---@param opponent BasePlacementOpponent
+---@return placement
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	lpdbData.weight = CustomPrizePool.calculateWeight(
+		lpdbData.prizemoney,
+		Variables.varDefault('tournament_liquipediatier'),
+		placement.placeStart
+	)
+
+	if opponent.opponentData.type == Opponent.solo then
+		lpdbData.mode = 'singles'
+		if opponent.additionalData.LASTVS then
+			lpdbData.extradata.lastvsflag = opponent.additionalData.LASTVS.players[1].flag
+		end
+	end
+
+	if opponent.opponentData.type == Opponent.duo then
+		lpdbData.mode = 'doubles'
+		lpdbData.players = {
+			p1 = opponent.opponentData.players[1].pageName,
+			p2 = opponent.opponentData.players[2].pageName,
+		}
```

Comment:
doesn't commons already set this (plus several other information in it)?

Reply:
Yup you're right. Also removed it from Brawlhalla, verified that it's not needed there